go.mod

```diff
@@ -11,7 +11,6 @@ replace github.com/golang/lint => golang.org/x/lint v0.0.0-20190301231843-5614ed
 replace (
 	cloud.google.com/go/pubsub => cloud.google.com/go/pubsub v1.3.1
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
-	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
```

Comment:
/hold
I suppose we may still need this then, I'm not sure what's pulling it in currently.

Reply:
`go mod vendor && grep -R docker vendor/` doesn't show any imports, maybe a workspace dependency ...